### PR TITLE
Add LoginPage and Header functionality

### PR DESCRIPTION
**Pull request by SebastianStupp**

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -31,13 +32,19 @@ const LoginInput = styled(Input)`
...
+  }
+
   return (
     <LoginContainer>
       <LoginLogo src={LogoLarge}></LoginLogo>
       <LoginTitle>LogiNicer</LoginTitle>
       <LoginInput placeholder="Username"></LoginInput>
       <LoginInput placeholder="Password"></LoginInput>
-      <Button>Login</Button>
+      <Button onClick={HandleOnClick}>Login</Button>
```
Comment (steinberggit) [inline client/src/pages/Login.js:47]: The button have no `cursor:pointe`r style.

File: client/src/components/Header.js [hunk 2]
```diff
@@ -35,14 +36,24 @@ const headerButtons = {
...
+    } else {
+      history.push('/menu');
+    }
+  }
+
   return (
     <Header>
       <HeaderImg src={MenuLogo} alt="MenuLogo" />
       <HeaderTitle>LogiNicer</HeaderTitle>
       <HeaderImg
```
Comment (steinberggit) [inline client/src/components/Header.js:53]: I miss the `cursor:pointer` style for the button

File: client/src/App.js [hunk 3]
```diff
@@ -14,6 +15,9 @@ function App() {
           <Route exact path="/">
             <Login></Login>
```
Comment (steinberggit) [inline client/src/App.js:16]: You could use` <Login />` instead of `<Login></Login>`
dito in line 19.

File: client/src/components/Header.js [hunk 4]
```diff
@@ -35,14 +36,24 @@ const headerButtons = {
 };
 
 export default function DefaultHeading(props) {
+  let location = useLocation();
+  let history = useHistory();
+  function HandleOnClick() {
+    if (location.pathname === '/menu') {
```
Comment (steinberggit) [inline client/src/components/Header.js:42]: I don't understand the if-else condition. Why did you need it? On the login page, I couldn't find this button.

Comment (SebastianStupp) [reply client/src/components/Header.js:42]: The button is the Logout Symbol in the Header :D

File: client/src/components/Header.js (hunk 2 above)
Comment (SebastianStupp) [reply client/src/components/Header.js:55]: I will add one :)

File: client/src/pages/Login.js (hunk 1 above)
Comment (SebastianStupp) [reply client/src/pages/Login.js:47]: I will add one :)

File: client/src/App.js (hunk 3 above)
Comment (SebastianStupp) [reply client/src/App.js:16]: Maybe I will refactor this one day, but at the moment my typing flow works better this way

File: client/src/components/Header.js (hunk 4 above)
Comment (SebastianStupp) [reply client/src/components/Header.js:44]: LoginPage has no Header, that's why u don't find it:) The menu page has Header with a logout symbol. But all other pages will have a header with u menu button to open the menu page :)
So the if-else condition is for the different Header types :)
